kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -350,11 +350,22 @@ public T apply(T resource) {
   }
 
   @Override
-  public T createOrReplace(T item) {
+  public T createOrReplace(T... items) {
+    T item = getItem();
```

Comment:
why do you store getItems() to item, if you are going to do nothing with it and replace it a few lines below? This lines seems uneeded.